### DISCUSSION TITLE
Enquiry on availability of PPG and blood glucose level data in MIMIC-IV

### DISCUSSION THREAD

**Discussion by Midhu7** [MIT-LCP/mimic-code#1343] [MIMIC-General]
Does MIMIC have the PPG (photoplethysmograph) time series data and Blood glucose level of patients?

**Comment by marymlucas** [comment]
Hi @Midhu7, 

The blood glucose levels should be available in the labevents table.

For the PPG -the waveform databases - https://physionet.org/content/mimic4wdb/0.1.0/ for MIMIC-IV and there's a corresponding one for MIMIC-III as well. They have both the waveforms and the numeric values

Mary
